docs/guides/managing_packages.md

```diff
@@ -0,0 +1,122 @@
+During the development of agents and services, users need to manage other packages, either developing them themselves or fetching already available ones. This guide will show you which are the relevant directories during development and how to get, use and publish packages.
```

Comment:
I am unsure if this has been defined anywhere else in the documentation, but perhaps this is a good place to define /remind what a package is in a couple of lines:

"Similarly as the Open AEA framework, the Open Autonomy framework also works with the concept of _pacakge_. Recall that a package is a collection of files that implement a specific component or functionality, and which are named as vendor/package_name:version."   

feel free to edit